easybuild/easyconfigs/l/libQGLViewer/libQGLViewer-2.8.0-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,21 @@
+name = 'libQGLViewer'
+version = '2.8.0'
+
+homepage = 'http://libqglviewer.com/'
+description = "libQGLViewer is a C++ library based on Qt that eases the creation of OpenGL 3D viewers."
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+source_urls = ['http://www.libqglviewer.com/src']
+sources = [SOURCE_TAR_GZ]
+checksums = ['03d2d3394866cdc419f43713aed827262c714cc4facddea7c3ba1d93dae3c4cc']
+
```

Comment:
please add builddependencies on binutils somewhere around here

Reply:
But for me it worked without binutils. Is it necessary?

Reply:
Might work for you, might not work on other systems. All GCCcore stuff should use EB's binutils.